docs/docusaurus.config.js

```diff
@@ -218,6 +218,24 @@ const config = {
       {
         fromExtensions: ["html"],
         toExtensions: ["html"],
+        createRedirects(existingPath) {
+          return [
+            existingPath.replace('/core', '/develop/advanced-concepts'),
+            existingPath.replace('/basics', '/develop/high-level-concepts'),
+            existingPath.replace('/intro', '/develop/intro'),
+            existingPath.replace('/architecture', '/integrate/architecture/'),
+            existingPath.replace('/building-apps', '/integrate/building-apps'),
+            existingPath.replace('/building-modules', '/integrate/building-modules'),
+            existingPath.replace('/tooling', '/integrate/tooling'),
+            existingPath.replace('/migrations', '/integrate/libraries'),
```

Comment:
This looks incorrect.

Reply:
`'/migrations', '/integrate/libraries'` looks incorrect :D, I think it should be `'/migrations', '/integrate/migrations'`

Reply:
lol yes haha